dateparser_scripts/write_complete_data.py

```diff
@@ -24,23 +24,27 @@
 RELATIVE_PATTERN = re.compile(r'\{0\}')
 
 
-def _modify_relative_data(relative_data):
+def _modify_relative_data(relative_data, replace_spaces=False):
     modified_relative_data = OrderedDict()
     for key, value in relative_data.items():
         for i, string in enumerate(value):
             string = RELATIVE_PATTERN.sub(r'(\\d+)', string)
+            if replace_spaces:
+                string = re.sub(r'\s+', '\\s?', string)
```

Comment:
So that we still allow more than 1 space character:

```suggestion
                string = re.sub(r'\s+', '\\s*', string)
```

Reply:
hmmmm... I thought it when coding, but the string is "sanitized" before, and multiple spaces are converted to just one, so the string shouldn't come with more than one space: https://github.com/scrapinghub/dateparser/blob/21c629f067a40c2becb19c7e88ec8170a8da3fcb/dateparser/date.py#L44-L48

You can try this in this branch and you will see that it's working:

```python
>>> dateparser.parse('2 分鐘前')
datetime.datetime(2021, 4, 29, 11, 9, 25, 525814)

>>> dateparser.parse('2  分鐘前')
datetime.datetime(2021, 4, 29, 11, 9, 27, 740355)

>>> dateparser.parse('2      分鐘前')
datetime.datetime(2021, 4, 29, 11, 9, 30, 363743)
```

I could change it, but maybe the intention is not so clear. What do you think?